subprojects/docs/src/docs/userguide/upgrading_version_5.adoc

```diff
@@ -176,9 +176,15 @@ Simply upgrade your IDE plugin to a version >= 1.3.50 to restore the correct Kot
 - Groovy has been updated to http://groovy-lang.org/changelogs/changelog-2.5.8.html[Groovy 2.5.8].
 - Kotlin has been updated to https://blog.jetbrains.com/kotlin/2019/08/kotlin-1-3-50-released/[Kotlin 1.3.50].
 
-The following breaking changes will appear as deprecation warnings with Gradle 5.6:
+==== Javadoc and Groovydoc don't include timestamps by default
+
+Timestamps in the generated documentation have very limited practical use, however they mark all the files as "modified" even in case of a small changes.
```

Comment:
"mark as modified" is not really well-defined here. The problem is (as you point out in the next sentence) is that timestamps (and also version numbers to some extent) make the build non-reproducible.

Reply:
How about now?